Makefile

```diff
@@ -140,7 +144,7 @@ verify-govulncheck:
 .PHONY: verify-sh
 verify-sh:
 	./scripts/fetch_tools.sh shellcheck && \
-	./_output/bin/shellcheck $$(find . -type d \( -path ./_output -o -path ./vendor -o -path ./assets -o -path ./etcd/vendor \) -prune -o -name '*.sh' -print)
+	./_output/bin/shellcheck $$(find . -type d \( -path ./_output -o -path ./vendor -o -path ./assets -o -path ./etcd/vendor \) -prune -o -name '*.sh' -print | grep -v -E "$(SHELLCHECK_IGNORE)")
```

Comment:
Or without resorting to grep (including suggestion from Giga)
```
verify-sh:
	IGNORE="configure-ovs*.sh" ; \
	./scripts/fetch_tools.sh shellcheck && \
	./_output/bin/shellcheck $$(find . \( -type d \( -path ./_output -o -path ./vendor -o -path ./assets -o -path ./etcd/vendor \) -o -name "$$IGNORE" \) -prune -o -name '*.sh' -print)
```
Although I agree (?) that reading `find` is not pleasant :laughing:

Reply:
Appreciate it! Updated